apps/rush-lib/src/cli/actions/ListAction.ts

```diff
@@ -34,7 +34,8 @@ export class ListAction extends BaseRushAction {
         'List package names, and optionally version (--version) and ' +
         'path (--path) or full path (--full-path), for projects in the ' +
         'current rush config.',
-      parser
+      parser,
+      suppressStartupBanner: true
```

Comment:
Can the startup banner only be suppressed if the `--json` flag is passed?

Reply:
I'd like to be able to use `rush list --to PROJECT | ForEach-Object { # Do stuff }` in PowerShell, instead of `rush list --to PROJECT --json | ConvertFrom-Json | % { # Do stuff with $_.name }`. More importantly, bash or batch scripts can work well with the former but not the latter.
Much of the intent of this feature was to avoid the "only suppressed if --json flag passed" logic and make it be the entire command having that decision.